packages/terra-demographics-banner/lib/DemographicsBanner.scss

```diff
@@ -0,0 +1,89 @@
+@import '~terra-mixins';
+@import './variables';
+
+.terra-DemographicsBanner {
+  background: $demographics-background;
+  color: $demographics-foreground;
+  display: flex;
+  flex-direction: row;
+  margin-bottom: 0.714em;
+  padding: 0.714em;
+
+  @media screen and (max-width: 767px) {
```

Comment:
I understand we have variables for these values inside Terra Legacy, but I'm finding that the 768 value set by that theme isn't working well with the ResponsiveElement. Is there a solution the rest of you guys would consider?